metashare/api/sf_run_flow.py

```diff
@@ -267,6 +250,12 @@ def create_org_and_run_flow(
     devhub_username = user.sf_username
     email = user.email  # TODO: check that this is reliably right.
```

Comment:
This seems like an odd assumption to me. I'd expect it to come from the active SF connection, not the (GitHub-created) `user.email`.

Reply:
We're deciding what email to request for the user that will be provisioned in the scratch org. Either one of these is a reasonable guess for something that is likely to reach the person who is creating the scratch org, but the one from github will work better in a scenario when multiple MS users are sharing the same dev hub user.